www-apps/hugo/hugo-0.125.4.ebuild

```
@@ -0,0 +1,101 @@
+# Copyright 2018-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module shell-completion
+
+DESCRIPTION="Fast static HTML and CSS website generator"
+HOMEPAGE="https://gohugo.io https://github.com/gohugoio/hugo"
+SRC_URI="
+	https://github.com/gohugoio/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://tastytea.de/files/gentoo/${P}-vendor.tar.xz
+"
+
+# NOTE: To create the vendor tarball, run:
+# `go mod vendor && cd .. && tar -caf ${P}-vendor.tar.xz ${P}/vendor`
+
+LICENSE="Apache-2.0 BSD BSD-2 MIT MPL-2.0 Unlicense"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~loong ~riscv ~x86"
```

```suggestion
KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
```
I'll fix this while merging

---

oops, thank you! 💖